_posts/2016-03-04-hunting-down-a-slow-rails-request.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Hunting Down a Slow Rails Request"
+date: 2016-03-04 11:03:00
+categories: ["rails", "performance"]
+author: "mauro-oto"
+---
+
+We have recently began using [Skylight](https://www.skylight.io) in production
+for one of our client's Rails applications, as an attempt to try to improve the
+performance of some of the more critical API endpoints.
+
+Skylight reports on both time taken, with a breakdown of time taken per SQL
```

Comment:
@mauro-oto This sounds a bit weird, could you refactor it to something like this? 

Skylight reports: 

* The time each request takes to complete
* A breakdown of the time it takes per SQL query
* Object allocations per request